### PR TITLE
docs: api/grn_proc: move grn_proc_get_info() reference to the header file

### PR DESCRIPTION
**Pull request by abetomo**
GitHub: GH-1817

Prepare to switch to documents automatically generated by Doxygen.

### CODE REVIEW

**Review by otegami — APPROVED** [review]
LGTM 👍🏾

File: include/groonga/proc.h [hunk 1]
```diff
@@ -64,6 +64,17 @@ grn_proc_create(grn_ctx *ctx,
                 grn_proc_func *fin,
                 unsigned int nvars,
                 grn_expr_var *vars);
+/**
+ * \brief Get information on procedure and expression that use `user_data`.
+ *
+ * \param ctx The context object.
+ * \param user_data The user_data.
```
Comment (kou) [inline include/groonga/proc.h:71]:
```suggestion
 * \param user_data The user data passed to \ref grn_proc_func.
```

File: include/groonga/proc.h [hunk 2]
```diff
@@ -64,6 +64,17 @@ grn_proc_create(grn_ctx *ctx,
...
+ * \param vars The buffer to store defined variables.
+ * \param nvars The buffer to store the number of `vars`.
+ * \param caller The buffer to store calling expression.
+ *
+ * \return The procedure object, but `NULL` if it is not set.
```
Comment (kou) [inline include/groonga/proc.h:76]:
```suggestion
 * \return The procedure object. It may be `NULL`.
```

**Review by otegami — APPROVED** [review]
LGTM 👍🏾

**Pull request merged**